CONTRIBUTING.md

```diff
@@ -44,5 +44,3 @@ following contribution guidelines:
 use of the pre-defined parent classes if appropriate
 
 3. Add the function to the namespace by adding it to `$GEOCAT_VIZ/src/geocat/viz/__init__.py`
-
-4. Add tests if appropriate in `$GEOCAT_VIZ/test/`
```

Comment:
Was this just removed temporarily as we do not have a test folder? Otherwise I think this should stay, but understand if that was the reasoning.

Reply:
That is why I took it out but I can put it back in if we want!

Reply:
Also I can't figure out how to comment on the suggestion above but it doesn't show that way for me when I load it on RTD - is there another way you are looking at it?